packages/starters/backend/sirius-components-task-starter/src/main/java/org/eclipse/sirius/components/task/starter/configuration/TaskExampleBuilder.java

```diff
@@ -67,11 +67,13 @@ private Project createDevProject(Person paul, Person peter) {
 
         Task idea = TaskFactory.eINSTANCE.createTask();
         idea.setName("Idea");
+        idea.setDescription("Description of the Idea");
```

Comment:
After a rebase on master, there is a problem with dates for Idea (start > end).
I think it's from the previous PR on deck

Reply:
not see this pb. I guess my last changes were not pushed yet to the upstream branch